express-api/src/controllers/notifications/notificationsController.ts

```diff
@@ -104,6 +107,40 @@ export const getNotificationInQueueById = async (req: Request, res: Response) =>
   return stubResponse(res);
 };
 
+/**
+ * @description Get all notifications for a specific project.
+ * @param {Request}     req Incoming request.
+ * @param {Response}    res Outgoing response.
+ * @returns {Response}      A 200 status with an object containing the specified notifications in the queue.
+ */
+export const getNotificationsByProjectId = async (req: Request, res: Response) => {
+  try {
+    const { projectId } = req.query;
+    const user = await userServices.getUser((req.user as SSOUser).preferred_username);
+
+    if (!projectId) {
+      return res.status(400).send({ message: 'Project ID is required.' });
+    }
+
+    // Fetch notifications in the queue for this project
+    const notificationFilter = {
+      projectId: Number(projectId), // Ensure projectId is a number
+      pageNumber: Number(req.query.pageNumber) || 0, // Default to page 0 if not provided
+      pageSize: Number(req.query.pageSize) || 0, // Default to 10 if not provided
+    };
```

Comment:
I think this could be handled more similarly to other controllers just for consistency sake, especially in regards to the key names for pageNumber and pageSize (elsewhere it is page and quantity).

Reply:
made some changes to this